src/students/example/mlflow/era5_temperature_project/era5_pipeline/assets.py

```
@@ -67,7 +66,7 @@ def raw_netcdf_dataset(
             "CDSAPI_URL and/or CDSAPI_KEY environment variables are not set. "
```

is this block (under "Check if CDSAPI_KEY and CDSAPI_URL are set, otherwise cdsapi.Client() might fail silently or use defaults") even needed? Are you sure it would fail silently?

---

Also, `CDSAPI_KEY` is not even the right name. It's `CDS_API_KEY`.. and `CDSAPI_URL` is not defined in github secrets either...

---

I think this block of code is dead code from before we started using ./run_dagster.sh or ev github secret for the api key

---

yes I'm changing it now